src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -100,7 +100,7 @@ object TSVFormatter {
     headerRenamingMap:Map[String, String]): IndexedSeq[String] = {
     val rowMap: Map[Int, String] =  entity.attributes map {
       case (attributeName, attribute) =>
-        val columnPosition = headerValues.indexOf(headerRenamingMap.getOrElse(attributeName.name, attributeName.name))
+        val columnPosition = headerValues.indexOf(attributeName.name)
```

Comment:
I am not sure this is right. If I am reading this right it is saying the header value (for a sample entity) has an entry for participant and we are not finding it because we are looking for participant_id. This code change makes it look for participant which allows us to find the data BUT the header should be participant_id. So this changes the wrong side of the equation. headerValues should have participant_id instead of participant.